README.md

```diff
@@ -85,6 +85,12 @@ Some rules require options. There are no default values, so each rule that requi
 }
 ```
 
+#### Shareable configs
+
+If you prefer to enforce a third-party styleguide (rather than craft your own config), you can use:
+
+* [SuitCSS shareable config](https://github.com/stylelint/stylelint-config-suitcss)
+
```

Comment:
I think so. I think that would be a pretty significant point for many users.

On Mon, Jun 29, 2015 at 10:16 AM, jeddy3 <notifications@github.com> wrote:

> In README.md
> <https://github.com/stylelint/stylelint/pull/255#discussion_r33487757>:
>
> > @@ -85,6 +85,12 @@ Some rules require options. There are no default values, so each rule that requi
> >  }
> >  ```
> >
> > +#### Shareable configs
> > +
> > +If you prefer to enforce a third-party styleguide (rather than craft your own config), you can use:
> > +
> > +* [SuitCSS shareable config](https://github.com/stylelint/stylelint-config-suitcss)
> > +
>
> Nice addition.
>
> There's an example of how to extend the config in the repo's readme (
> https://github.com/stylelint/stylelint-config-suitcss#extending-the-config).
> Should be copy over that example into the main project readme, or is it
> fine as it is?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stylelint/stylelint/pull/255/files#r33487757>.
>